src/components/home/elements/SendToken.vue

```diff
@@ -0,0 +1,960 @@
+            type: String,
+            default: '',
+            require: true
+        },
+        tokenBalances: {
+            type: Object,
+            default: function() {
+            },
+            require: true
+        },
+        tokenUnity: {
+            type: BigNumber,
+            default: function() {
+            },
+            require: true
+        },
+        balances: {
+            type: Object,
+            default: function() {
+            },
+            require: true
+        },
+        functionIndex: {
+            type: Number,
+            default: SEND_FUNCIDX,
+            require: true
+        }
+    },
+    data: function() {
+        return initData
+    },
+    created() {
+        this.coldRecipientAddressList = new LRUCache(10)
+        this.hotRecipientAddressList = new LRUCache(10)
+        let item = window.localStorage.getItem('Cold ' + this.defaultColdAddress + ' sendTokenRecipientAddressList ')
+        if (item) {
+            this.coldRecipientAddressList.load(JSON.parse(item))
+        }
+        item = window.localStorage.getItem('Hot ' + this.defaultAddress + ' sendTokenRecipientAddressList ')
+        if (item) {
+            this.hotRecipientAddressList.load(JSON.parse(item))
+        }
+    },
+    computed: {
+        contractId() {
+            return transaction.tokenIDToContractID(this.tokenId)
+        },
+        defaultAddress() {
+            return Vue.ls.get('address')
+        },
+        defaultColdAddress() {
+            if (this.noColdAddress) return ''
+            return Object.keys(this.coldAddresses)[0]
+        },
+        userInfo() {
+            return JSON.parse(window.localStorage.getItem(this.defaultAddress))
+        },
+        secretInfo() {
+            return JSON.parse(
+                seedLib.decryptSeedPhrase(this.userInfo.info, Vue.ls.get('pwd')))
+        },
+        seedPhrase() {
+            return seedLib.decryptSeedPhrase(this.secretInfo.encrSeed, Vue.ls.get('pwd'))
+        },
+        wordList() {
+            return this.seedPhrase.split(' ')
+        },
+        isSubmitDisabled() {
+            return !(this.recipient && BigNumber(this.amount).isGreaterThan(0) && this.isValidRecipient(this.recipient) && (this.isValidAttachment || !this.attachment) && this.isAmountValid('hot') && this.address !== '')
+        },
+        isColdSubmitDisabled() {
+            return !(this.coldAddress && this.coldRecipient && this.coldAmount > 0 && this.isValidRecipient(this.coldRecipient) && (this.isValidColdAttachment || !this.coldAttachment) && this.isAmountValid('cold') && this.coldAddress !== '')
+        },
+        noColdAddress() {
+            return Object.keys(this.coldAddresses).length === 0 && this.coldAddresses.constructor === Object
+        },
+        dataObject() {
+            return {
+                protocol: PROTOCOL,
+                api: this.coldApi(),
+                opc: OPC_FUNCTION,
+                address: this.coldAddress,
```

Comment:
it may correct, but it seems use `this.address` in issue and burn's code. it will be confused to use different settings.

Reply:
The issue and burn token modals cannot select address but sendToken can. Now sendToken using `this.address` for hot wallet address and `this.coldAddress` for cold wallet address. `Send.vue` and `SendToken.vue` will be refactored later and only use 1 set of variables and functions for both hot wallet and cold wallet in later version.